sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/implementation/SparkBridgeImplementationInternal.scala

```diff
@@ -294,4 +294,8 @@ private[cosmos] object SparkBridgeImplementationInternal extends BasicLoggingTra
       None
     ).json()
   }
+
+  def configureSimpleObjectMapper(allowDuplicateProperties: Boolean) : Unit = {
+    Utils.configureSimpleObjectMapper(true)
```

Comment:
```suggestion
    Utils.configureSimpleObjectMapper(allowDuplicateProperties)
```

Reply:
No - allowing duplicate properties won't just impact a single Cosmos Client but is a global setting. Called this out in the config description. So once you enable it in one config dictionary it will be enabled forever for the lifecycle of the process.

Reply:
Fixed